hello-static.yml

```diff
@@ -0,0 +1,41 @@
+application:
+  interfaces:
+    input:
+      "*": bind(workflow#input.*)
+    output:
+      message: bind(workflow#result.message)
+  components:
+    workflow:
+      type: workflow.Instance
+      interfaces:
+        input:
+          ip: configuration(string)
+        result:
+          message: publish-signal(map<string, string>)
+      configuration:
+        input.ip: 127.0.0.1
+        configuration.workflows:
+          launch:
+            steps:
+              - provision:
+                  action: provisionVms
+                  phase: provisioning
```

Comment:
I'd say that explicit `phase` declaration is needed too rarely to be included in the basic hello-world